pages/eval.js

```diff
@@ -8,15 +8,26 @@ import Rubric from '../components/Evaluator/Rubric';
 import { getAllApplicants, getHackathons } from '../utility/firebase';
 
 const Container = styled.div`
-  display: flex;
+  display: grid;
+  grid-template-columns: 1fr 1.5fr 1fr;
+  grid-template-rows: 1fr;
+  gap: 1rem;
+  height: 100vh;
+  box-sizing: border-box;
+  padding: 0 1rem;
+  background: #f7f7f7;
 `;
 
 const Column = styled.div`
-  flex: 1.5;
-`;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 1rem 0;
```

Comment:
Not sure if this makes more or less sense, but does combining the 1rem top+bottom padding from this with the left+right padding in `Container` (line 17) make sense? Assuming that the padding is defined by the Container and not by the Column

Reply:
This makes a lot more sense TY